### PR TITLE
docs(README): explain `outputPath` && `publicPath` subtility

### PR DESCRIPTION
**Pull request by jdeniau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
...
+  loader: 'file-loader',
+  options: {
+    outputPath: 'images/'
+  }  
+}
+```
+
+Your files will be written in `public/images/` and your browser will require `/images/foo.png`
+
+If you add a `"publicPath": "/foo/"`, you files will still be writter in `public/images/` folder but your browser will require `/foo/images/foo.png`.
```
Comment (drbonzo) [inline README.md]: Typos spotted:

```
- you files will still be writter
+ your files will still be written
```

File: README.md [hunk 2]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
 
 You can specify custom `output` and `public` paths by using `outputPath`, `publicPath` and `useRelativePath`
 
+`outputPath` is relative to your webpack `output.path` configuration.
```
Comment (michael-ciniawsky) [inline README.md]: ``` `outputPath` is relative to `output.path` specified in your configuration (`webpack.config.js`) ```

File: README.md [hunk 3]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
...
+
+`publicPath` is "added" in front of your url.
```
Comment (michael-ciniawsky) [inline README.md]: ``` `publicPath` is appended to the URL of the file  ```

File: README.md [hunk 4]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
 
 You can specify custom `output` and `public` paths by using `outputPath`, `publicPath` and `useRelativePath`
 
+`outputPath` is relative to your webpack `output.path` configuration.
+
+`publicPath` is "added" in front of your url.  
+
+Example:
```
Comment (michael-ciniawsky) [inline README.md]: `Example:`  => `**webpack.config.js**`

File: README.md [hunk 5]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
...
+
+{
+  loader: 'file-loader',
+  options: {
+    outputPath: 'images/'
+  }  
+}
+```
+
+Your files will be written in `public/images/` and your browser will require `/images/foo.png`
```
Comment (michael-ciniawsky) [inline README.md]: ``` Your files will be written to `public/images/` and served to the browser from `/images` (e.g ``) ```

File: README.md [hunk 6]
```diff
@@ -115,6 +115,26 @@ By default, the path and name you specify will output the file in that same dire
...
+
+If you add a `"publicPath": "/foo/"`, your files will still be written in `public/images/` folder but your browser will require `/foo/images/foo.png`.
```
Comment (michael-ciniawsky) [inline README.md]: ``` If you override the `publicPath` via `publicPath: '/foo/'` (`options.publicPath`), your files will still be written to `public/images/`, but  served to the browser as `/foo/images/file.png` ```

**Comment by phy25** [comment]
What's the status of this PR? By the way, @jdeniau it seems that a CLA is required to sign according to https://cla.js.foundation/webpack-contrib/file-loader?pullRequest=202.

**Comment by jdeniau** [comment]
@phy25 Sorry for the CLA, it's signed.

For the status of this PR, it's "just" some explanation in the README, so unless there have been some major changes in the loader, it should still be OK.

**Comment by michael-ciniawsky** [comment]
Closing due to inactivity for now, feel free to reopen :)

**Pull request closed without merging**